app/models/container_label_tag_mapping.rb

```diff
@@ -91,9 +91,18 @@ def self.create_tag(name, value, category_tag)
       entry_name = Classification.sanitize_name(value)
       description = value
     end
-    entry = category.add_entry(:name => entry_name, :description => description)
-    entry.save!
-    entry.tag
+    upsert_tag(category, entry_name, description)
   end
   private_class_method :create_tag
+
+  def upsert_tag(category, name, description)
+    entry = category.add_entry(:name => name, :description => description)
+    if entry.valid?
```

Comment:
Yes it is.  DIscussion moved to https://github.com/ManageIQ/manageiq/pull/11806/files#r82697923 which is similarly racy.